### PR TITLE
[#670] Institutionalize Department of Psychiatry and Behavioral Sciences

### LINKED ISSUE #670: Create Institutional Collection: Department of Psychiatry and Behavioral Sciences
Thanks @kglibrarian for the details.

The Department of Psychiatry and Behavioral Sciences 
proposed name of the collection- Department of Psychiatry and Behavioral Sciences Collection
a short description- A digital collection of information from the Department of Psychiatry and Behavioral Sciences
some keywords- psychiatry, psychology, psych, brain, addiction, neuromodulation, neuropsychology, forensic psychiatry, consultation-liaison psychiatry, geriatric psychiatry, behavioral health, women's behavioral health, psychiatric education
names and NetIDs of collection owners- Rachel Rodriguez (rrr998)
visibility level of the collection- Open access
 
They have also asked for two sub-collections (and 4 sub-sub-collections) within their collection. Below are the details:

- Department Events
  - Conferences
  - Commencement
  - Grand Rounds
  - Scholars Day

- Newsletters

### PR DESCRIPTION
**Pull request by sharpattack**
Add sub collections
- Department Events
  - Conferences
  - Commencement
  - Grand Rounds
  - Scholars Day

- Newsletters

Closes #670

### CODE REVIEW

File: lib/batch/institutionalize_psychiatry_and_behavorial_sciences.rb [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+dept_of_psych_and_behavioral_sci_sys_user = User.create!(
+  username: 'institutional-department-of-psychiatry-and-behavioral-sciences-system',
+  email: 'institutional-department-of-psychiatry-and-behavioral-sciences-system@northwestern.edu',
+  display_name: "Department of Psychiatry and Behavioral Sciences"
+)
+
+role = Role.create!(name: admin_group)
+
+#collection_user = User.find_or_create_via_username("rrr998")
+collection_user = User.find_or_create_via_username("gso1135")
```
Comment (fenekku) [inline lib/batch/institutionalize_psychiatry_and_behavorial_sciences.rb]: Shouldn't we be using `rrr998` here?

File: lib/batch/institutionalize_psychiatry_and_behavorial_sciences.rb [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+collection_user.save!
+
+# The parent collection
+dept_of_psych_and_behavioral_sci_col = Collection.new(
+  title: 'Department of Psychiatry and Behavioral Sciences Collection',
+  tag: ["psychiatry", "psychology", "psych", "brain", "addiction",
+        "neuromodulation", "neuropsychology", "forensic psychiatry",
+        "consultation-liaison psychiatry", "geriatric psychiatry",
+        "behavioral health", "women's behavioral health",
+        "psychiatric education"],
```
Comment (fenekku) [inline lib/batch/institutionalize_psychiatry_and_behavorial_sciences.rb]: These tags can be set to a variable and re-used throughout.

File: lib/batch/institutionalize_psychiatry_and_behavorial_sciences.rb (hunk 1 above)
Comment (sharpattack) [reply lib/batch/institutionalize_psychiatry_and_behavorial_sciences.rb]: Yes, good catch.

File: lib/batch/institutionalize_psychiatry_and_behavorial_sciences.rb (hunk 2 above)
Comment (sharpattack) [reply lib/batch/institutionalize_psychiatry_and_behavorial_sciences.rb]: Okay, done.

**Pull request merged**